src/bin/rdcore/kargs.rs

```diff
@@ -19,9 +19,21 @@ use libcoreinst::io::*;
 #[cfg(target_arch = "s390x")]
 use libcoreinst::s390x;
 
+#[cfg(target_arch = "s390x")]
+use libcoreinst::blockdev::*;
+
 use crate::cmdline::*;
 use crate::rootmap::get_boot_mount_from_cmdline_args;
 
+#[cfg(target_arch = "s390x")]
+pub fn zipl(config: ZiplConfig) -> Result<()> {
```

Comment:
I'd prefer we put this at the end of the file rather than the start since it's a "guest" here.